### ISSUE TITLE
Doesn't work with symfony/console v5.1.1

### ISSUE THREAD

**Issue by SmetDenis** [phpbench/phpbench#638]
Hi there,

First of all, it's really nice project. Super great work! Awesome.

Unfortunately, I have an issue with the last version `symfony/console@5.1.1` which is released yesterday.

I suppose it's a small issue with the type of return variable. Should be `NULL` instead of `0`

See log here - https://travis-ci.org/github/JBZoo/Path/jobs/697937749#L493

### Commands
```sh
php `pwd`/vendor/bin/phpbench run         \
        --config="$(JBZOO_CONFIG_PHPBENCH)"    \
        --tag=jbzoo                            \
        --warmup=2                             \
        --store                                \
        --stop-on-error

# It fails here =(
php `pwd`/vendor/bin/phpbench report      \
        --config="$(JBZOO_CONFIG_PHPBENCH)"    \
        --uuid=tag:jbzoo                       \
        --report=summary                       \
        --precision=2
```

### Config file
```json
{
    "bootstrap"        : "vendor/autoload.php",
    "path"             : "tests/phpbench",
    "xml_storage_path" : "build/phpbench/storage",
    "progress"         : "travis",

    "outputs"          : {
        "jbzoo-html-summary" : {
            "extends" : "html",
            "file"    : "build/phpbench/summary.html"
        },
        "jbzoo-md"           : {
            "extends" : "markdown",
            "file"    : "build/phpbench/for-readme.md"
        }
    },

    "reports"          : {
        "jbzoo"   : {
            "generator" : "table",
            "title"     : "Summary benchmark info (execution time)",
            "break"     : ["benchmark"],
            "cols"      : [
                "benchmark",
                "subject",
                "groups",
                "its",
                "revs",
                "mean",
                "stdev",
                "rstdev",
                "mem_real",
                "diff"
            ],
            "sort"      : {
                "mean"      : "asc",
                "benchmark" : "asc"
            }
        },
        "summary" : {
            "generator" : "composite",
            "reports"   : ["jbzoo", "env"]
        }
    }
}
```

### Error message
```
PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Return value of "PhpBench\Console\Command\ReportCommand::execute()" must be of the type int, "null" returned. in /home/travis/build/JBZoo/Path/vendor/symfony/console/Command/Command.php:261
Stack trace:
#0 /home/travis/build/JBZoo/Path/vendor/symfony/console/Application.php(911): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#1 /home/travis/build/JBZoo/Path/vendor/symfony/console/Application.php(264): Symfony\Component\Console\Application->doRunCommand(Object(PhpBench\Console\Command\ReportCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /home/travis/build/JBZoo/Path/vendor/symfony/console/Application.php(140): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /home/t in /home/travis/build/JBZoo/Path/vendor/symfony/console/Command/Command.php on line 261
Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Return value of "PhpBench\Console\Command\ReportCommand::execute()" must be of the type int, "null" returned. in /home/travis/build/JBZoo/Path/vendor/symfony/console/Command/Command.php on line 261
Symfony\Component\Debug\Exception\FatalThrowableError: Return value of "PhpBench\Console\Command\ReportCommand::execute()" must be of the type int, "null" returned. in /home/travis/build/JBZoo/Path/vendor/symfony/console/Command/Command.php on line 261
Call Stack:
    0.1025   10142456   1. Symfony\Component\Debug\ErrorHandler->handleException() /home/travis/build/JBZoo/Path/vendor/symfony/debug/ErrorHandler.php:0
```

**Comment by dantleech** [comment]
Thanks, fixed and tagged 0.17.1 :)

**Comment by SmetDenis** [comment]
Oh... I didn't know. Thank you!
